src/action/config/ActionConfiguration.cpp

```diff
@@ -304,15 +309,27 @@ action::Action::Timing ActionConfiguration::getTiming() const
   PRECICE_TRACE(_configuredAction.timing);
   action::Action::Timing timing;
   if (_configuredAction.timing == VALUE_REGULAR_PRIOR) {
-    timing = action::Action::ALWAYS_PRIOR;
+    timing = action::Action::WRITE_MAPPING_PRIOR;
+    PRECICE_WARN("Regular-prior action timings will no longer be supported. Regular-prior will now revert to Write-mapping-prior which performs "
+                  "the action timings before writeMappingData and before the coupling update.");
```

Comment:
For consistency with the documentation, more user-friendly terms:
```suggestion
    PRECICE_WARN("Regular-prior action timings will no longer be supported. Regular-prior will now revert to write-mapping-prior which performs "
                  "the action before a write mapping and before the coupling update.");
```